<?xml version="1.0"?>
<root main_tree_to_execute="BehaviorTree">
    <!-- ////////// -->
    <BehaviorTree ID="BehaviorTree">
        <Sequence>
            <Action ID="MovePlat2" position="1"/>
            <Action ID="MovePlat1" position="1"/>
            <Action ID="MovePlat1" position="0"/>
            <Action ID="MovePlat2" position="0"/>
        </Sequence>
    </BehaviorTree>
    <!-- ////////// -->
    <TreeNodesModel>
        <Action ID="MovePlat1">
            <input_port name="position"/>
        </Action>
        <Action ID="MovePlat2">
            <input_port name="position"/>
        </Action>
        <Action ID="MovePlat3">
            <input_port name="position"/>
        </Action>
        <Action ID="MovePlat4">
            <input_port name="position"/>
        </Action>
        <Condition ID="WaitAndCheckStatus"/>
    </TreeNodesModel>
    <!-- ////////// -->
</root>

